modules/component-store/src/component-store.ts

```diff
@@ -152,6 +152,17 @@ export class ComponentStore<T extends object> implements OnDestroy {
     }
   }
 
+  protected get<R>(projector: (s: T) => R): R {
```

Comment:
Should we make it easier to access the whole state? 
We could allow `get()`, and it would return the state.

```ts
protected get<R>(projector?: (s: T) => R): R {
	projector = projector || (s => s)
	...
}
```

Reply:
Done 👍